src/java/com/bean/ListFolderBean.java

```diff
@@ -33,13 +62,24 @@ public void setUser(User user) {
         this.user = user;
     }
     
-   public List<Folder> getListFolder() throws Exception{
+    public int getTotalPage() throws Exception {
+        if(size % pageSize == 0) return size / pageSize;
+        else return 1 + size / pageSize;
+    }
+    
+   public List<Folder> getListfolder() throws Exception{
+             List<Folder> list = new ArrayList<>();
+             
+                if(page == 0) page = 1;
+                if(pageSize == 0) pageSize = 5;
+                int from = (page - 1) * pageSize;
+                int to = page * pageSize;
             if(user == null){
-                return Folder.getListFolder(null, -1, 3);
+               list = Folder.getListFolder(null, -1, 3);
 
                  }
             else{
-                List<Folder> list = new ArrayList<>();
+              
```

Comment:
- Add get List Folder for admin
- Wrong params in getListFolder when user is logged in.